sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1124,6 +1129,29 @@ public enum JsonPropertyNames
         UseExact = 0,
         CamelCase = 1,
     }
+    public partial class ModelJsonConverter : System.Text.Json.Serialization.JsonConverter<Azure.Core.Serialization.IModelSerializable>
+    {
+        public ModelJsonConverter() { }
+        public ModelJsonConverter(bool ignoreAdditionalProperties) { }
+        public bool IgnoreAdditionalProperties { get { throw null; } set { } }
+        public override bool CanConvert(System.Type typeToConvert) { throw null; }
+        public override Azure.Core.Serialization.IModelSerializable Read(ref System.Text.Json.Utf8JsonReader reader, System.Type typeToConvert, System.Text.Json.JsonSerializerOptions options) { throw null; }
+        public override void Write(System.Text.Json.Utf8JsonWriter writer, Azure.Core.Serialization.IModelSerializable value, System.Text.Json.JsonSerializerOptions options) { }
+    }
+    public static partial class ModelSerializer
+    {
+        public static T Deserialize<T>(System.BinaryData data, Azure.Core.Serialization.ModelSerializerOptions? options = null) where T : class, Azure.Core.Serialization.IModelSerializable { throw null; }
+        public static System.BinaryData Serialize<T>(T model, Azure.Core.Serialization.ModelSerializerOptions? options = null) where T : class, Azure.Core.Serialization.IModelSerializable { throw null; }
+    }
+    public partial class ModelSerializerOptions
+    {
+        public ModelSerializerOptions() { }
+        public bool IgnoreAdditionalProperties { get { throw null; } set { } }
+        public bool IgnoreReadOnlyProperties { get { throw null; } set { } }
+        public string? NameHint { get { throw null; } set { } }
+        public bool PrettyPrint { get { throw null; } set { } }
```

Comment:
Would it make sense to change this to an enum, to let customer have finer grained control of this?  It would be good to provide samples for use-case scenarios for this, to motivate its need and name a bit more.

Reply:
this was removed